docs/development/doc_guide.rst

```diff
@@ -426,6 +432,34 @@ substitutions should not be used in docstrings when it is important that
 users have quick access to the full path of the `object` (such as in the
 ``See Also`` section).
 
+Bibliography
+~~~~~~~~~~~~
+
+PlasmaPy uses |sphinxcontrib-bibtex|_ to manage references. This
+extension allows us to store references in a BibTeX_ file which is then
+used to generate the :doc:`../bibliography`.
```

Comment:
```suggestion
PlasmaPy uses |sphinxcontrib-bibtex|_ to manage references for its
documentation. This Sphinx_ extension allows us to store references
in a BibTeX_ file which is then used to generate the 
:doc:`../bibliography`.  References in the :doc:`../bibliography` are then
citable from anywhere in the documentation.
```

I mainly just wanted to get the last sentence in there.  Feel free to play with the wording though.